src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  //for bg image to fill the whole screen
+  //background-position: center;
+  //background-size: cover;
+  background-color: lightblue;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    box-sizing: border-box;
+    height: 100vh;
+    padding: $headerTopPadding 0 80px;
+  }
+
+  &__top {
+    @include top-actions;
+  }
+
+  &__title {
+    font-size: 46px;
+    line-height: 40px;
+    font-weight: bold;
+    color: #fff;
+    letter-spacing: -1px;
+
+    &::before {
+      content: "";
+      display: block;
+      width: 40px;
+      height: 2px;
+      background-color: #fff;
```

Comment:
Again, consider using an SCSS variable for the color value instead of the hardcoded value '#fff'. You can use the same variable as before, e.g. $headerTextColor. 🎨